web/Dockerfile

```diff
@@ -1,11 +1,23 @@
-FROM nginx:mainline-alpine
+FROM nginx
+
+# install cron, supervisor and certbot
+RUN echo "deb http://ftp.debian.org/debian stretch-backports main" | tee -a /etc/apt/sources.list
+RUN apt-get update
+RUN apt-get install -qy cron supervisor python-certbot-nginx -t stretch-backports
```

Comment:
Is it possible to use a separate container instead of supervisor?

Reply:
Do we need an extra container?